src/Util/MailSender.php

```diff
@@ -73,6 +83,7 @@ public function sendEmailMessage(
             ->setSubject($emailTwigTemplate->renderBlock('subject', $mailData))
             ->setFrom($this->from)
             ->setTo($toAddresses)
+            ->setBcc($this->bccAddress)
```

Comment:
What happens if this is null, blank, not set? Error, or just no e-mail?

Reply:
well, .env is checked into git and contains a default, so is this still a concern?

Reply:
I don't think there would be a scenario where it would not be set, the Constructor of MailSender expects a string and it is by default set in the .env file. I tried with empty string, there is no email sent